diff --git a/node_modules/@ant-design/react-native/react-native.config.js b/node_modules/@ant-design/react-native/react-native.config.js
index db45665..4215284 100644
--- a/node_modules/@ant-design/react-native/react-native.config.js
+++ b/node_modules/@ant-design/react-native/react-native.config.js
@@ -1,10 +1,9 @@
 module.exports = {
-  dependency: {
-    assets: ['../icons-react-native/fonts'],
-  },
   project: {
+    ios: {},
     android: {
-      sourceDir: './rn-kitchen-sink/android',
-    },
+      sourceDir: './rn-kitchen-sink/android'
+    }
   },
+  assets: ['../icons-react-native/fonts']
 };
